src/website/views/components/DropDown/DropDown.tsx

```diff
@@ -0,0 +1,90 @@
+import React from 'react';
+import { Accordion, AccordionSummary, AccordionDetails } from '@mui/material';
+import ExpandMoreIcon from '@mui/icons-material/ExpandMore';
+import { styled } from '@mui/material/styles';
+
+const textAndTitleColor = '#092e4e';
+
+const FlexItemContainer = styled('div')({
+  flex: 1,
+  display: 'flex',
+  justifyContent: 'center',
+  fontFamily: 'Switzer, sans-serif',
+  color: textAndTitleColor,
+  fontWeight: 'bold',
+});
+
+const DetailsFlexContainer = styled('div')({
+  display: 'flex', 
+  justifyContent: 'space-between',
+  width: '100%',
+  position: 'relative', 
+  paddingBottom: '8px',
+  marginTop: '2px',
+  '&::after': { 
+    content: '""', 
+    display: 'block',
+    position: 'absolute',
+    bottom: '0', 
+    left: '0', 
+    width: '100%', 
+    height: '1px', 
+    backgroundColor: 'black', 
+  },
+});
+
+const ContentContainer = styled('div')({
+  display: 'flex',
+  justifyContent: 'space-between',
+  width: '100%',
+  paddingTop: '8px',
+});
```

Comment:
Could we have everything in this file dealing with styles added to a `.css` file? This implementation isn't wrong, just deals with our style guidelines. 

Depending on the project, it can go with either styled components or with classnames, but it's not good to mix them.